ScoutSuite/output/data/html/partials/aws/services.rds.regions.id.vpcs.id.instances.html

```diff
@@ -9,8 +9,7 @@ <h4>Information</h4>
           <ul>
             <li class="list-group-item-text">Region: {{region}}</li>
             <li class="list-group-item-text">Engine: {{Engine}}</li>
-            <li class="list-group-item-text">Status: {{make_title DBInstanceStatus}}</li>
-            <li class="list-group-item-text">Is read replica: {{is_read_replica}}</li>
+            <li class="list-group-item-text">Status: {{makeTitle DBInstanceStatus}}</li>
```

Comment:
I think the `is_read_replica` line was lost in a merge somewhere. 
```suggestion
            <li class="list-group-item-text">Is read replica: {{is_read_replica}}</li>
            <li class="list-group-item-text">Status: {{makeTitle DBInstanceStatus}}</li>
```

Reply:
Not merged yet here but it is on our broken merging branch!